src/__tests__/multiStore.test.js

```diff
@@ -30,6 +51,11 @@ describe('multiStore', () => {
       sessionStorage.setItem('foo', 'baz')
       expect(stores.get('foo')).toBe('baz')
     })
+    test('calls the error handler if an error is thrown', () => {
+      errorStores.get('foo')
+      expect(errorHandler).toBeCalledTimes(1)
+      expect(errorHandler).toHaveBeenCalledWith(errorToThrow, 'foo')
```

Comment:
You can combine these with [`toHaveBeenNthCalledWith`](https://jestjs.io/docs/en/expect#tohavebeennthcalledwithnthcall-arg1-arg2-) 😬

Reply:
Nice, I'll use it in the other PR too 😄 Thank you 🙂